### PR TITLE
Add pycurses TUI and copy to clipboard (closes #1)

### LINKED ISSUE #1: Add usage of pycurses with keyboard keys (arrows) and enter
Like in title. Highlitght actually chosen link and enter to copy videos url to clipboard.

### PR DESCRIPTION
**Pull request by jonasfs**

### CODE REVIEW

File: src/flyt.py [hunk 1]
```diff
@@ -19,33 +21,66 @@ def validateArgument(argument):
...
-def getLatestVideosList(argument, length):
+def get_video_list(argument, length):
+    video_list = list()
     ytname = validateArgument(argument)
     header = {'user-agent': 'Mozilla/3.0', 'Accept-Language': 'en-US,en;q=0.5'}
     r = requests.get(ytname, headers=header)
+
     if r.status_code == 200:
-        prettify(r.text, length)
+        soup = BeautifulSoup(r.text, 'html.parser')
```
Comment (jakubbogucki) [inline src/flyt.py]: Create function out of it.

Comment (jakubbogucki) [inline src/flyt.py]: Please create function out of it.

**Comment by jonasfs** [comment]
Can you tell me which distro and terminal emulator you're using? And do you happen to have xclip installed? Clipboards are really tricky to work with. I never had to use the alternative dependencies of pyperclip (I use xclip), but it says [here](https://github.com/asweigart/pyperclip) that it should work too with xsel, or by importing pygtk or pyqt4 on the project. What do you think would be the best?

The background color is black on all the terminals I tested too (tty, st, konsole, xterm, guake, tilda), I think its supposed to be same background color of the terminal by default. Should I force it to be black?

File: src/flyt.py (hunk 1 above)
Comment (jonasfs) [reply src/flyt.py]: You mean from the r.status_code check onwards?

**Comment by jakubbogucki** [comment]
It is okay then, maybe I messed something up on my terminal. :)  Great thanks for your help.

**Comment by jonasfs** [comment]
No worries, if anything comes up let me know (or open an issue), I'm actually curious if theres a better way to do this too!

**Pull request merged**